tmt/options.py

```diff
@@ -279,6 +281,12 @@ def _check_method(self, context: 'tmt.cli.Context', args: List[str]) -> None:
                 raise tmt.utils.SpecificationError(
                     f"Unsupported {self.name} method '{how}'.")
 
+        def is_likely_subcommand(self, arg: str, subcommands: List[str]) -> bool:
+            for subcommand in subcommands:
+                if subcommand.startswith(arg):
+                    return True
+            return False
```

Comment:
1. it's probably not necessary for this code to be a method, it can be a nice, small function, even a pure one as its return value depends on its inputs and nothing else, no `self`, no `cls`.
2. and it can be reduced with the help of `any` :)

```python
def is_likely_subcommand(arg: str, subcommands: List[str]) -> bool:
    return any(subcommand.startswith(arg) for subcommand in subcommands)
```

Reply:
Hi @happz, thank you for your quick review.  I have updated the related code according to your comments, please help to take a look again, thanks!

Reply:
LGTM, thanks.